### PR TITLE
[xcode13.3] Bring Xcode 13.3 support

### PR DESCRIPTION
**Pull request by dalexsoto**
This PR brings Xcode 13.3 support up to Beta 3.

Probably better reviewed commit by commit.

### CODE REVIEW

File: Versions-ios.plist.in [hunk 1]
```diff
@@ -48,6 +48,7 @@
 			<string>14.5</string>
 			<string>15.0</string>
 			<string>15.2</string>
+			<string>15.4</string>
```
Comment (rolfbjarne) [inline Versions-ios.plist.in:51]: What happened to 15.3?

File: src/GameController/GCMotion.cs [hunk 2]
```diff
@@ -58,4 +59,58 @@ public struct GCQuaternion {
...
+		const int DiscretePositionCount = 10; // From GCDualSenseAdaptiveTrigger.h
+		[MarshalAs (UnmanagedType.ByValArray, SizeConst = DiscretePositionCount)]
+		public float [] Values;
+
+		public GCDualSenseAdaptiveTriggerPositionalResistiveStrengths (float [] values)
+		{
+			if (values is null)
+				ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (values));
+			if (values.Length > DiscretePositionCount)
+				throw new ArgumentException ($"The '{nameof (values)}' array length can't be greater than {DiscretePositionCount}.");
```
Comment (rolfbjarne) [inline src/GameController/GCMotion.cs:111]: Can it be less that DiscretePositionCount?

**Review by rolfbjarne — CHANGES_REQUESTED** [review]
Looks good, only missing tests for the manual bindings :)

File: Versions-ios.plist.in (hunk 1 above)
Comment (dalexsoto) [reply Versions-ios.plist.in:51]: No idea, what is shipped on Xcode 13.3 is 15.4 I wonder if they are moving to even numbers releases just like us, it seems that there is not 15.1 either heh

File: src/healthkit.cs [hunk 3]
```diff
@@ -128,6 +128,38 @@ public enum HKCategoryValueProgesteroneTestResult : long {
 		Indeterminate,
 	}
 
+	[Watch (8,5), iOS (15,4), MacCatalyst (15,4)]
+	public enum HKVerifiableClinicalRecordSourceType {
+		[DefaultEnumValue]
+		[Field (null)]
```
Comment (tj-devel709) [inline src/healthkit.cs:134]: Nice 👍

**Review by tj-devel709 — APPROVED** [review]

File: src/healthkit.cs [hunk 4]
```diff
@@ -3718,8 +3760,17 @@ interface HKVerifiableClinicalRecordQuery
...
+		[BindAs (typeof (HKVerifiableClinicalRecordSourceType []))]
+		[Export ("sourceTypes", ArgumentSemantic.Copy)]
+		NSString [] SourceTypes { get; }
+
 		[Export ("initWithRecordTypes:predicate:resultsHandler:")]
 		NativeHandle Constructor (string[] recordTypes, [NullAllowed] NSPredicate predicate, HKVerifiableClinicalRecordQueryResultHandler handler);
+
+		[iOS (15,4)]
+		[Export ("initWithRecordTypes:sourceTypes:predicate:resultsHandler:")]
+		IntPtr Constructor (string [] recordTypes, [BindAs (typeof (HKVerifiableClinicalRecordSourceType []))] NSString [] sourceTypes, [NullAllowed] NSPredicate predicate, Action<HKVerifiableClinicalRecordQuery, HKVerifiableClinicalRecord [], NSError> resultsHandler);
```
Comment (mandel-macaque) [inline src/healthkit.cs:3773]: Lets use a delegate for such a large action, two params and an error.

**Review by mandel-macaque — APPROVED** [review]
Small nits to be fixe before landing, yet it looks good.

File: src/avfoundation.cs [hunk 5]
```diff
@@ -10753,6 +10789,16 @@ interface AVCaptureDevice {
 		[Export ("exposurePointOfInterest")]
 		CGPoint ExposurePointOfInterest { get; set;  }
 
+		[NoWatch]
+		[NoTV, NoMac, MacCatalyst (15,4), iOS (15,4)]
```
[the remark this answers is not in the document]
Comment (dalexsoto) [reply src/avfoundation.cs:10793]: Just to follow along the style in the file, see methods above

**Comment by dalexsoto** [comment]
Bringing in another PR

**Pull request closed without merging**